packages/website/pages/explore/index.tsx

```diff
@@ -21,59 +23,84 @@ const GET_NFTS = gql`
 `;
 
 const ExploreToken = () => {
-  const { loading, data, error, fetchMore } = useQuery(GET_NFTS, {
-    variables: {
-      offset: 0,
-      limit: LIMIT_PER_PAGE,
-    },
-  });
+  const [getNfts, { loading, data, error, fetchMore }] = useLazyQuery(
+    GET_NFTS,
+    {
+      variables: {
+        offset: 0,
+        limit: LIMIT_PER_PAGE,
+      },
+    }
+  );
+
+  useEffect(() => {
+    getNfts();
+  }, []);
 
   return (
     <Layout>
-      {loading && <Spinner />}
-      {error && <p>Error: check console</p>}
-      {!loading && !error && (
-        <Feed
-          entries={data.nfts || []}
-          onLoadMore={() =>
-            fetchMore({
-              variables: {
-                offset: data.nfts.length,
-              },
-            })
-          }
-        />
-      )}
+      <Box sx={{ textAlign: "center", my: 30, mx: "auto", maxWidth: 900 }}>
+        {loading && <Spinner />}
+        {error && <p>Error: check console</p>}
+        {!loading && !error && (
+          <Feed
+            entries={data?.nfts || []}
+            onLoadMore={() =>
+              fetchMore({
+                variables: {
+                  offset: data.nfts.length,
+                },
+              })
+            }
+          />
+        )}
+      </Box>
     </Layout>
   );
 };
 
 const Feed = ({ entries, onLoadMore }) => {
   return (
-    <>
-      <button onClick={onLoadMore}>load more</button>
-
-      <InfiniteScroll dataLength={entries.length} next={onLoadMore}>
-        {entries.map((item, index) => {
-          return (
+    <InfiniteScroll dataLength={entries.length} next={onLoadMore}>
```

Comment:
Hi, change `LIMIT_PER_PAGE` to less than 6. Then it should load remaining `NFTs`. But it is not loading them. I think here `dataLength` should be the length of all `NFTs`. Not the Length of fetched `NFTs`.

Reply:
Thanks for pointing that out, infinite loading wasn't working at all. I think It's fixed now with the last commit